notes/notes.md

```diff
@@ -31,9 +31,10 @@
 
 ### Start a demo runtime
 
-* Open the project "org.eclispe.smarthome.demo.app"
+* Open the project "org.openhab.core.demo.app"
 * Open the file "app.bndrun"
 * Press "resolve" to resolve the "run requirements" and ensure that all requirements are able to satisfied.
+* Make sure, that the Compiler Compliance Level in Eclipse matches the installed JDK: set it to 8, if there is Java 8 installed
```

Comment:
AFAIK openHAB Core -- as it depends on Xtext < 2.17 -- does not support any Java version > 8 at all.

Reply:
Yes, was just a reminder, because for me, installed only Java 8, the runtime did not start, because Eclipse has set compiler compliance level to 10... However, @kaikreuzer mentioned a better fix in the setup file: 
``` 
<setupTask
          xsi:type="setup:PreferenceTask"
          key="/instance/org.eclipse.jdt.core/org.eclipse.jdt.core.compiler.compliance"
          value="1.8"/>
```